packages/primitives/progress-bar/readme.md

```diff
@@ -0,0 +1,17 @@
+`ProgressBar` is a box that holds other components inside. It can be styled to some limited extent. The closest analogs are:
+
+- `div` in Web
+- `View` in Native
+
+```ts
+type TBlock<TStyles> = {
```

Comment:
Should be `type TProgressBar<TSt...`, right?

Reply:
copy and..... ffffuuuuuuuuuu!
will fix

Reply:
done